README.md

```diff
@@ -1,2 +1,97 @@
 # c11-poseidon-earthquake-monitoring
-This project will monitor the United States Geological Survey (USGS) earthquake data feeds, continually extracting and storing data. This data will be used to provide interactive dashboards  and subscribe-able alerts, so that users can make appropriate preparations/decisions with full knowledge of earthquake risks.
+This project will monitor the United States Geological Survey (USGS) earthquake data feeds, continually extracting and storing data. This data will be used to provide interactive dashboards  and subscribable alerts, so that users can make appropriate preparations/decisions with full knowledge of earthquake risks.
+
+## Folders
+
+| Folder Name | Description |
+|---|---|
+| **api** | An API to allow developers to query earthquake data. |
+| **database** | A database to store all the real-time earthquake data. |
+| **diagrams**  | Images that map out the project for easy understanding. |
+| **.github** | Essential files for your GitHub repository. |
+| **pipeline**  | Code that brings data from the QuakeML API to the database. |
+
+## Architecture Design
+
+For a visual representation of the project architecture and details on the design decisions, please refer to this section.
+
+### Architecture Diagram
+
+![Architecture Diagram](TO BE FILLED)
+
+### Design Decisions
+
+## Database Design
+
+For a visual representation of the database schema and details on the design decisions, please refer to this section.
+
+### Entity-Relationship Diagram (ERD)
+
+![ERD Diagram](TO BE FILLED)
+
+### Design Decisions
+
+
+## Getting Setup
+
+The sections below detail all the instructions to get this project running.
+
+### Dependencies
+
+1. **Clone the repository**:
+    ```bash
+    git clone https://github.com/danfh00/vodnik-plant-health-monitor.git
+    ```
+
+2. **Create and activate a virtual environment (optional but recommended)**:
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
```

Comment:
```suggestion
2. **Create and activate a virtual environment (optional but recommended)**:
    ```bash
    python3 -m venv .venv
    source .venv/bin/activate
    pip install -r requirements.txt
    ```
```

Might be worth adding a comment that this will need to be done inside each individual folder